keyboards/handwired/hexon38/hexon38.c

```diff
@@ -0,0 +1,3 @@
+// see https://github.com/pepaslabs/hexon38
+
+#include QMK_KEYBOARD_H
```

Comment:
This shoud actually be the keyboard name. It can cause issues with revisions, if it isn't. 

```suggestion
 #include "hexon38.h"
```